ui/utils.py

```diff
@@ -152,3 +152,8 @@ def get_bucket(bucket_name):
 def get_transcoder_client():
     """Get an ElasticTranscoder client object"""
     return boto3.client('elastictranscoder', settings.AWS_REGION)
+
+
+def get_s3_client():
+    """Get an S3 client object"""
+    return boto3.client('s3')
```

Comment:
where are you using this? if the only place is in the watch API, you can move the code there, otherwise please add tests and a better docstring for this.